pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/multifile/ApexMultifileAnalysis.java

```diff
@@ -92,8 +92,10 @@ public final class ApexMultifileAnalysis {
             // found, so we get a java.nio.file.ProviderNotFoundException. Since all this happens during initialization of the class
             // com.nawforce.apexlink.types.platform.PlatformTypeDeclaration we get a ExceptionInInitializerError
             // and later NoClassDefFoundErrors, because PlatformTypeDeclaration couldn't be loaded.
-            LOG.error("Exception while initializing Apexlink ({})", e.getMessage(), e);
-            LOG.error("PMD will not attempt to initialize Apexlink further, this can cause rules like UnusedMethod to be dysfunctional");
+            if(LOG.isErrorEnabled()) {
```

Comment:
not only does this code not comply with our code style (missing whitespace after `if`), this is pointless. The code is using placeholders, and therefore doesn't perform expensive operations before being short-circuited by SLF4J. This is in compliance with our own [`GuardLogStatementRule`](https://docs.pmd-code.org/latest/pmd_rules_java_bestpractices.html#guardlogstatement)